script/Arcueid/Day 2/02_00_ARC02_5.txt

```diff
@@ -6,18 +6,19 @@ It's ten o'clock.
 [sha:1284ddfb99e96e68bc6dcf921fb143346f948d20]{
 -- Page 10018, Offset 3469.
 -- 俺が消すまでもなく、電灯は自動的に消灯した。
-// Can someone check the JP text to see which of these is closer to the original meaning? I might be being a little too liberal with it. Does need the comma, though.
-I don't need to turn off the lights, as they turn off automatically.
+The lights turn themselves off automatically, without me having to do anything.
 }
 [sha:ddbd3def31dd03a67363da505304243f21f84ecc]{
 -- Page 10018, Offset 3470.
 -- この時間になると、執務室以外の明かりはいったん落ちる仕組みなのだそうだ。
-I heard that at this time, all of the lights except for those in the office are turned off.
+Apparently all of the lights are programmed to switch off at this hour,
```

Comment:
that last clause after the comma is kinda awkward. I think dropping the last clause in the middle is a little smoother.
> Seems like all of the lights, except those in the office, are programmed to switch off at this hour